docs/faq/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _faq:
+
+****
+FAQ
+****
+
+We've assembled some caveats and gotchas in this Frequently Asked
```

Comment:
I know I said FAQ is good earlier on slack... But I guess now I'm not so sure given this (quite sensible) layout: it's not really "questsions".  More like "Frequently Useful Stuff", I guess...?  Not sure I have a better suggestion... But my pedantic nature twitches at a "Questions" section that isn't a bunch of questions...